src/pool/srv_pool.c

```diff
@@ -1086,8 +1286,18 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 			D_ERROR(DF_UUID": failed to read pool map buffer: "
 				""DF_RC"\n", DP_UUID(svc->ps_uuid), DP_RC(rc));
 		}
+		D_GOTO(unlock, rc);
+	}
+
```

Comment:
Would you mind removing this blank line, please? The comment on line 1274 (after applying this PR) is supposed to cover the block ending at line 1304. Thanks in advance.

If we feel this block has become too long, we could extract this TX to a separate function.

Reply:
ok